control/init.sh

```diff
@@ -132,6 +132,10 @@ if ! grep -q init-ansible ~/.profile 2>/dev/null; then
             echo '[ -t 1 ] && source ~/init-ansible' >> ~/.profile
             printf "${YELLOW}→ Added init script to ~/.profile\n"
         ;;
+        [Nn]* )
+            echo '# [ -t 1 ] && source ~/init-ansible' >> ~/.profile
```

Comment:
if someone explicitly says no, adding a line, even a commented one, seems like the wrong behavior.

Reply:
Yeah that's true. The thing is that without adding this commented out line, it'll all you every single time. I wanted to make it easy for people to say no once and then never be asked about it again. I had the control machine in mind but I guess some people install it locally using this script as well? I'm open to changing this.